syntax = "proto3";
package meterusage;

import "google/protobuf/timestamp.proto";


service MeterUsageService {
  rpc GetMeterUsage(MeterUsageRequest) returns (stream MeterUsageResponse) {}
}

message MeterUsageRequest {

}

message MeterUsageResponse {
  google.protobuf.Timestamp time = 1;
  double meterusage = 2; // Using float was causing precision error
}